test/testcafe/spec/EnrollProfileView_spec.js

```diff
@@ -80,3 +87,17 @@ test.requestHooks(requestLogger, EnrollProfileSignUpWithAdditionalFieldsMock)('s
   await t.expect(await enrollProfilePage.isDropdownVisible('userProfile.timezone')).ok();
   await enrollProfilePage.selectValueFromDropdown('userProfile.timezone', 1);
 });
+
+test.requestHooks(requestLogger, EnrollProfileSignUpWithBooleanFieldsMock)('should show radio and checkbox display for boolean data type fields', async t => {
+  const enrollProfilePage = new EnrollProfileViewPageObject(t);
+  const identityPage = await setup(t);
+  await identityPage.clickSignUpLink();
+
+  requestLogger.clear();
+
+  await t.expect(await enrollProfilePage.clickRadioButton('userProfile.pet', 0)).eql('Yes');
+
+  await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.subscribe')).eql(false);
+  await enrollProfilePage.setCheckbox('userProfile.subscribe');
```

Comment:
Does this line click the checkbox? Can we maybe verify it's been clicked after with another 
`await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.subscribe')).eql(true);`

Reply:
good question. i did the debug mode, and set did the check. And i also added `await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.subscribe')).eql(true);`  but it is still eql to `false` I have not figured out why.

Reply:
🤔 that's strange, does the `setCheckbox` method take any other params?

Reply:
there is a para `value`, but i do not think it helps (also tested). All get do is 
`return this.el.find(```input[name="${name}"]```).checked;` 
and in ui, it did check. so I have no clue =\